plugins/airbrake/src/api/AirbrakeApi.ts

```diff
@@ -24,3 +24,9 @@ export const airbrakeApiRef = createApiRef<AirbrakeApi>({
 export interface AirbrakeApi {
   fetchGroups(projectId: string): Promise<Groups>;
 }
+
+export class NoProjectIdError extends Error {
+  constructor() {
```

Comment:
Best use `CustomErrorBase` from `@backstage/errors` to make sure this error behaves as one would expect